Jenkinsfile

```diff
@@ -13,14 +13,14 @@ pipeline {
     stage('Validate') {
       parallel {
         stage('Changelog') {
-          steps { sh './bin/parse-changelog.sh' }
+          steps { sh './tools/parse-changelog.sh' }
```

Comment:
@diverdane looking at this now, wdyt about calling the dir `ci` instead of `tools`? I'm not sure people will think to look here for these scripts in the future, but we do use `ci` in other projects, so that may be more consistent.

on the other hand, as long as this is documented in CONTRIBUTING, it should be fine :)

Reply:
@izgeri, I like that suggestion! The scripts in that directory are all called by the Jenkinsfile, so it makes sense.